zubbi/doc.py

```diff
@@ -88,8 +97,8 @@ def render_sphinx(content):
 
             # Add the mocked SupportedOS directive to get the os information
             # without rendering it into the resulting HTML page
-            app.add_directive("supported_os", SupportedOS)
-
+            app.add_directive(SUPPORTED_OS_DIRECTIVE_NAME, SupportedOS)
```

Comment:
Maybe we could ditch the `*_DIRECTIVE_NAME` constants and directly use `SupportedOS.directive_name` or `Reusable.directive_name` here

Reply:
I am not sure if the directive_name can be accessed via class name, I will try.